### ISSUE TITLE
`chia check`: is it normal for not all proofs to be successful?

### ISSUE THREAD

**Issue by matteomonti** [Chia-Network/chia-blockchain#691]
After some weeks of plotting, I decided to run a `chia check` just to make sure that all was good. To my surprise, I got a stream of error messages saying that some of the plots were invalid. 

Suspecting that failure could be probabilistic, I added a `-n 64` parameter to the command: now all plots seem to be valid, but none of them passes all tests. The fraction of tests that each plot passes, weirdly enough, seems to be in the range from 0.75 to actually more than 1 (I saw up to 1.21). Is this normal? 

In summary, which one of the two tests should I believe? Was the first test telling me that some plots were invalid only because by chance they got 0 success? And, is there such a thing as a plot (which I produced straight out of the GUI) being partially compromised, or produced somewhat partially improperly?

**Comment by hoffmang9** [comment]
The testing is uses random challenges so it's going to bounce around. Anything in that range (0.75 to 1.25) is fine and normal. The more tests you use, the more statistically valid the test is. 1000 will likely show the range narrow considerably - but is not quick.

**Comment by danielritchie** [comment]
Hi @hoffmang9 - what about when it's outside of that range?  

I have a plot that says "Found 1 valid plots" but is pretty consistently Proofs 19 / 30, 0.6333.  Does this mean it's bad and I should trash it?
